modules/gitops-installing-argo-rollouts-cli-by-using-linux.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-cli-on-linux_{context}"]
+= Installing Argo Rollouts CLI on Linux
+
+If you are a Linux user, you can install Argo Rollouts CLI by performing the steps described in this procedure.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (`oc`)
+
+.Procedure
+
+. Download the latest version of the Argo Rollouts CLI binary, `kubectl-argo-rollouts`, by running the following command:
++
+[source,terminal]
+----
+$ curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+----
+
+. Ensure that the `kubectl-argo-rollouts` binary is executable by running the following command:
++
+[source,terminal]
+----
+$ chmod +x ./kubectl-argo-rollouts-linux-amd64
+----
+
+. Move the `kubectl-argo-rollouts` binary to the system path by running the following command:
++
+[source,terminal]
+----
+# mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
+----
+
+[IMPORTANT]
+====
+Ensure that you have superuser privileges to run this command.
+====
+
+. Verify that the plugin is installed correctly by running the following command and receiving the similar output:
```

Comment:
This could be the start of a `.Verification` section.

Reply:
This was a verification option earlier but as per the feedback received, I have added it into a step. Kindly advise on how to proceed.

Reply:
I wasn't aware that GitOps didn't use that Verification style. It's a standard part of mod docs. 

https://redhat-documentation.github.io/modular-docs/#verification

If there's really a prohibition against it that is specific to GitOps, yeah, don't use it.

Reply:
No, as far as I know, there are no issues with using the verification style. It is just modified as per the review feedback received earlier.